src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.Api.sync.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
+    {
+        public static T FromJson<T>(string json, JsonConverterSettings options = null)
+        {
+            return (T)FromJson(json, typeof(T), options);
+        }
+
+        public static object FromJson(string json, Type returnType, JsonConverterSettings options = null)
+        {
+            byte[] jsonBytes = Encoding.UTF8.GetBytes(json);
+            return FromJson((ReadOnlySpan<byte>)jsonBytes, returnType, options);
```

Comment:
nit: `jsonBytes.AsSpan()` reads better than the cast, imo

Reply:
Apparently there is an implicit cast operator from byte[] to ReadOnlySpan, so I can just remove that cast.